### ISSUE TITLE
Deploy preview in PRs

### ISSUE THREAD

**Issue by evenstensberg** [mi6/ic-ui-kit-test-app#27]
I need feedback on this, but is automated deploy previews of this application wanted? It will simplify testing PRs, as well as you will have a domain to check/verify logic without having to run the test application locally.

CC @GCHQ-Developer-530

**Comment by evenstensberg** [comment]
I'd suggest [Netlify](https://www.netlify.com/), it's british and is ok+. Vercel is also a good alternative.

**Comment by GCHQ-Developer-530** [comment]
It would be better using something like Github pages like we have done for our ic-ui-kit and ic-design-system repos

**Comment by GCHQ-Developer-530** [comment]
Yeah it should be the `ic-ui-kit-deploy` step in `ic-ui-kit-branches.yml` 👍

**Comment by ec1467** [comment]
We've decided to not do this. We have this on some of the stackblitzes on the site, and it would increase our consumption of GitHub resources.
